homeassistant/components/notify/html5.py

```diff
@@ -131,29 +140,79 @@ def post(self, request):
         return self.json_message("Push notification subscriber registered.")
 
 
+class HTML5PushCallbackView(HomeAssistantView):
```

Comment:
Where would we pass this? Under a notify platform?

This won't work for iOS, as we need to pre-define actions. The way I currently do this is by configuring actions under the ios component, and then exposing them over HTTP for the iOS app to consume and process into the format that Apple accepts them in. I think that iOS is the only notify platform with this style of setup though, so a generic actions implementation could be built but `html5` would be the only platform taking advantage of it for the time being.